src/SamplesApp/SamplesApp.UITests.Generator/CategoryBucketGenerator.cs

```diff
@@ -56,7 +51,7 @@ where typeSymbol.GetMembers().OfType<IMethodSymbol>().Any(m => m.FindAttributeFl
 			{
 				var builder = new IndentedStringBuilder();
 
-				builder.AppendLineInvariant("using System;");
+				builder.AppendLineIndented("using System;");
```

Comment:
We were using `IndentedStringBuilder` from `Uno.RoslynHelpers` package. Now, we unify this with other generators that are using `IndentedStringBuilder` defined here in this repo.